TheFourFifths/consus-client
# Editing a model.

### Summary

You can edit a model by clicking the page/pen icon on the view all models screen.

### Checklist


### How to Review

1. Start server on edit-model-tff111 OR edit-model-story if merged already
2. Start client on this branch
3. View all models and click the pen(cil) to edit the model
4. Witness the form and input any information you feel like editing
5. Verify toast
6. View all models and checkout your changes(does not change the address)

### Links

- [TFF-111](https://msoese.atlassian.net/browse/TFF-111)
- TheFourFifths/consus/pull/41
